repos/system_upgrade/el7toel8/models/saphanainfo.py

```diff
@@ -0,0 +1,35 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+
+class SapHanaInstanceInfo(Model):
+    """ Details of SAP HANA instances """
+    topic = SystemInfoTopic
+
+    name = fields.String()
+    """ Name of the instance """
+    path = fields.String()
+    """ Directory of the SAP HANA instance """
+    instance_number = fields.String()
+    """ SAP HANA Instance number """
+    running = fields.Boolean()
+    """ Is the instance is currently running? """
+    admin = fields.String()
+    """ Name of the instance administrator """
+    manifest = fields.JSON()
```

Comment:
@vinzenz  I guess you used the JSON because we are not sure what all values could be in the manifest? Otherwise it could be represented e.g. by SapHanaManifest model with the specified fields. We are strict about the use of JSON type. let's sync. but I prefer to replace it if we know what could be inside and it's not tooo long list.

Reply:
some manifest examples are in one of the tests, not sure if it is worth making it a model...

Reply:
@Rezney  we are relatively strongly against the JSON models and we require people to write the correct models instead. I prefer to not have an exception if not needed. This one seems to be easy to cover by model.

Reply:
But if we do now know whole manifest - so it's problematic to cover correctly in all possible outputs - it's ok to keep json, at least for now.

Reply:
There are changing values in the manifest, and I wanted to keep it completely for debugging purposes.
We do not know the whole manifest and there are keys that can change from release to release.
e.g. 
```
max_rel2.0_sps0_rev-number: 02
max_rel2.0_sps0_patchlevel: 02
max_rel2.0_sps1_rev-number: 12
max_rel2.0_sps1_patchlevel: 05
max_rel2.0_sps2_rev-number: 24
max_rel2.0_sps2_patchlevel: 10
max_rel2.0_sps3_rev-number: 37
max_rel2.0_sps3_patchlevel: 07
max_rel2.0_sps4_rev-number: 48
max_rel2.0_sps4_patchlevel: 02
```

I change change it for something more strict.